meeting-notes/2024-06-20.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: '2024-06-20'
+authors: naman
+tags: [developer]
+---
+
+<iframe
+  src="https://drive.google.com/file/d/1nrEQdhiTH4wsQVgaiOO0RcmD-AS-5S2v/preview"
+  width="640"
+  height="360"
+  allow="autoplay"
+></iframe>
+
+1. [Kirckz discusses Meru](https://docs.google.com/presentation/d/1Fu4AkB0mrvOkK6UDFJHgKwCV-Ul4JRF-xPqTYJ3CQqw), a Financial services app for Freelancers and remote workers in Latin America.
+2. He shares his experience integrating Meru with Blend, a liquidity protocol primitive for Stellar.
+3. Kirckz shares the challenges faced during the integration and how they were overcome.
+4. He shares the sdks and libraries his team used to facilitate the integration.
+
+Follow Meru on Twitter to stay updated: https://x.com/getmeru
```

Comment:
Should we do the whole `X (formerly Twitter)` thing? 🤣 

(I vote sticking with just Twitter lol)

Reply:
Oh snap! 

Yeah I can push a change rn